pkg/controllers/dynakube/extension/otel/statefulset_test.go

```diff
@@ -282,7 +302,8 @@ func TestAnnotations(t *testing.T) {
 		statefulSet := getStatefulset(t, getTestDynakube())
 
 		assert.Len(t, statefulSet.ObjectMeta.Annotations, 1)
-		assert.Empty(t, statefulSet.Spec.Template.ObjectMeta.Annotations)
+		assert.Len(t, statefulSet.Spec.Template.ObjectMeta.Annotations, 1)
```

Comment:
please use `require.Len` to check existence and length of Annotations map and then `assert.<Condition>` to safely verify the contents of the map

Reply:
Fixed!